DependencyInjection/MonologExtension.php

```diff
@@ -923,6 +947,7 @@ private function getHandlerClassByType($handlerType)
             'debug' => 'Symfony\Bridge\Monolog\Handler\DebugHandler',
             'swift_mailer' => 'Symfony\Bridge\Monolog\Handler\SwiftMailerHandler',
             'native_mailer' => 'Monolog\Handler\NativeMailerHandler',
+            'symfony_mailer' => 'Symfony\Bridge\Monolog\Handler\MailerHandler',
```

Comment:
same here, not possible to use the ::class ?
it ease IDE refactor/search etc :)